libraries/node-core-library/src/JsonSchema.ts

```diff
@@ -94,6 +111,11 @@ export interface IJsonSchemaLoadOptions {
    * or does not match an expected URL, the default version will be used.
    */
   schemaVersion?: JsonSchemaVersion;
+
+  /**
+   * Any custom formats to consider during validation.
```

Comment:
Flesh this doc comment out + provide some more info on a use-cases for this for devs